test/serializers/default_include_test.rb

```diff
@@ -0,0 +1,52 @@
+require 'test_helper'
+
+module ActiveModel
+  class Serializer
+    class DefaultIncludeTest < ActiveSupport::TestCase
+      class Blog < ActiveModelSerializers::Model
+        attr_accessor :id, :name, :posts
+      end
+      class Post < ActiveModelSerializers::Model
+        attr_accessor :id, :title, :author
+      end
+      class Author < ActiveModelSerializers::Model
+        attr_accessor :id, :name
+      end
+      class BlogSerializer < ActiveModel::Serializer
+        attributes :id
+        attribute :name, key: :title
+
+        has_many :posts
+      end
+      class PostSerializer < ActiveModel::Serializer
+        default_include [:author]
+        attributes :id, :title
+
+        has_one :author
```

Comment:
Just as a background, `include` as an association option was already discussed on #1113, but finally dismissed in favour of a controller-level include.
There were many reasons for these (I strongly advise a quick read on the above issue), but it was mainly due to the confusion this could produce.

I think that `default_include` and `always_include` would provide a useful additional feature, without getting too much out of context and without creating confusion.

It's really just a simple way to keep code DRY and avoid copy-pasting includes in your controllers when the same serialiser is used.

This doesn't mean that we can't implement also an association-level `include`, I see it as a complementary feature, and not a substitution for this one 😄